glide.yaml

```diff
@@ -79,3 +79,4 @@ import:
 - package: github.com/jaytaylor/html2text
 - package: gopkg.in/gomail.v2
   version: 2.0.0
+- package: github.com/ssor/bom
```

Comment:
I can fork the html2text and remove the bom library and write my own without using external lib.